Automattic/jetpack
# VideoPress: serve a minified version for the token bridge file in production

This PR generates a minified version of the `videopress-token-bridge`.js file to be served in the production environment.
Also and on the fly, re-organize the built files structure.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: serve a minified token bridge version in production env
* VideoPress: re-organize built files

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Enable the VideoPress module
* Go to wp-admin
* Confirm `videopress-token-bridge.js` file is minified in production env. Use a JN site
* Confirm `videopress-token-bridge.js` is not minified in development env.

dev | prod
------|------
<img width="776" alt="image" src="https://user-images.githubusercontent.com/77539/185205887-8b297c16-b6bc-4112-8b77-6226f803f11e.png"> | <img width="778" alt="image" src="https://user-images.githubusercontent.com/77539/185205737-99d08fb4-d2eb-4f78-bdb1-2057393ec7d2.png">


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202764755494536
  - https://app.asana.com/0/0/1202798948032541
